locales/en/messages.json

```diff
@@ -1224,22 +1224,22 @@
         "message": "Benewake LIDAR"
     },
     "pidTuningUpgradeFirmwareToChangePidController": {
-        "message": "<span class=\"message-negative\">Changing PID controller disabled - you can change it via the CLI.</span>  You have firmware with API version <span class=\"message-negative\">$1</span>, but this functionality requires <span class=\"message-positive\">$2</span>."
+        "message": "<span class=\"message-negative\">Changing PIDF controller disabled - you can change it via the CLI.</span>  You have firmware with API version <span class=\"message-negative\">$1</span>, but this functionality requires <span class=\"message-positive\">$2</span>."
```

Comment:
Not sure if we actually want to change how we call the controller in user facing text.
'PID controller' is a well established term (https://en.wikipedia.org/wiki/PID_controller), 'PIDF controller' is not. In fact, 'PIDF controller' seems to be used for a model that is 'PID + filter' (https://ieeexplore.ieee.org/document/7358695/), which is different from what we are doing.

Reply:
Yes, that was one of my doubts. This was discussed in slack and it seemed accepted, but if not, I can change it here and in the blackbox PR, both of them change the PID concept to PIDF concept.